.github/workflows/dockerhub.yml

```diff
@@ -8,9 +8,27 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: trigger
-      run: curl -X POST https://hub.docker.com/api/build/v1/source/1813a660-2ee5-4583-a238-dd54e9a6ebac/trigger/c8cd9f1f-f269-45bc-9750-a08327257f62/call/
-    
-    
-
-
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: install buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: latest
+      - name: login to docker hub
+        run: |
+          echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+      - name: build and push the image (master branch)
+        run: |
+          docker buildx build \
+            --push \
+            --tag neilpang/acme.sh:latest \

```

Comment:
please use `${{github.ref}}`  instead of `latest` here.

Reply:
`${{github.ref}}` will have something like `refs/master` are you sure that is what you want here?

Reply:
sorry, it was my mistake. it's ok.